hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientListenerServiceImpl.java

```diff
@@ -160,11 +160,15 @@ private ClientEventProcessor(Packet packet) {
 
         @Override
         public void run() {
-            final ClientConnection conn = (ClientConnection) packet.getConn();
-            final ClientResponse clientResponse = serializationService.toObject(packet.getData());
-            final int callId = clientResponse.getCallId();
-            final Data response = clientResponse.getResponse();
-            handleEvent(response, callId, conn);
+            ClientConnection conn = (ClientConnection) packet.getConn();
+            ClientResponse clientResponse = serializationService.toObject(packet.getData());
```

Comment:
Can this line (or one of the others) throw an exception? Should we maybe put those 4 lines into the try-finally block like in the legacy client?